src/components/Run.js

```diff
@@ -1,17 +1,21 @@
-import React from "react";
-import Radium from "radium";
-import Button from "./Button";
+import React from 'react';
+import Radium from 'radium';
+import Button from './Button';
+import Spinner from './Spinner';
+import {styles} from './styles';
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import {run} from '../redux/actionCreators';
-import {styles} from './styles';
 
 export class Run extends React.Component {
   render() {
-    if (this.props.running) {
+    if (this.props.running && this.props.isLoadedRuntime) {
       return (
         <Button kind="run" style={{color: "gray", paddingTop: 12}}>
-          <img style={styles.gif} src={this.props.gif}/>
+          <Spinner
+            indicator="https://code.pyret.org/img/pyret-spin.gif"
```

Comment:
can we make this prop optional? Or get rid of it all together? I don't thin we need to be able to pass in different indicators.